### ISSUE TITLE
system_interrupt.h line 426

### ISSUE THREAD

**Issue by BriscoeTech** [adafruit/Adafruit_ASFcore#2]
Somehow someone decided to comment out this bracket and it is causing compiler errors in the arduino ide

The comment needs to be restored to prevent compiler errors

**Comment by jaromy** [comment]
Agreed. I also found this bug while trying to compile in Arduino IDE 1.8.0. Thanks for finding and posting this issue, BriscoeTech. It definitely needs to be fixed. Library will not compile as-is!

**Comment by JetForMe** [comment]
I also have this problem, over a year later.
